tests/test_configuration.py

```diff
@@ -0,0 +1,64 @@
+from mock import MagicMock, mock_open
```

Comment:
From https://docs.python.org/3.5/library/unittest.mock.html#unittest.mock.mock_open:

> Changed in version 3.5: read_data is now reset on each call to the mock.

We need this otherwise a second read would fail.